packages/teams-js/src/public/search.ts

```diff
@@ -1,8 +1,20 @@
-import { sendMessageToParent } from '../internal/communication';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import {
+  Communication,
+  sendAndHandleStatusAndReason as sendAndHandleError,
```

Comment:
I know we do this import _bla_ as sendAndHandlerError in other places in the code, but it would be good to just import sendAndHandleStatusAndReason directly instead of aliasing it. Aliasing it confuses other developers and helps disguise bugs.

Reply:
Sound good will change it